wiki/Ranking_Criteria/en.md

```diff
@@ -85,29 +86,36 @@ Note: If a mapset's song contains matching sections to the TV size song, in the
 
 ### Rules
 - **No two hit objects can be placed on the same tick** This includes hit circles, starts and ends of sliders, and starts and ends of spinners. osu!mania maps are exempt from this.
```

Comment:
Add a line between the header and text. Do for every instance.

Add a period after the bold statement.

`maps` -> `beatmaps` Do for every instance. Part of the ASC. Find and replace also `map`, the noun.

Reply:
applied both